qualification/report/generate_pdf.py

```diff
@@ -886,9 +913,11 @@ def main():
     result_list = list_from_json(args.input)
     if args.commit_id:
         print(test_image_commit(result_list))
-    doc = document_from_list(result_list, args.report_doc_id)
+    procedure_doc, report_doc = document_from_list(result_list, args.report_doc_id, "E1200-0000-004")
```

Comment:
Minor compared to other observations, but perhaps declare these document number constants elsewhere. It looks particularly odd here where you've got the cmd-line arg with a default, then this 'hardcoded' arg here.